lib/src/package.dart

```diff
@@ -191,6 +191,7 @@ class Package {
     'packages/',
     'pubspec.lock',
     '!pubspec.lock/', // We allow a directory called pubspec lock.
+    'pubspec_overrides.yaml',
```

Comment:
I think we should do
```suggestion
    '/pubspec_overrides.yaml',
```

To only ignore the file at the root (not deeper in the hierarchy).